invenio_communities/assets/semantic-ui/js/invenio_communities/members/Filters.js

```diff
@@ -54,7 +55,7 @@ export class Filters {
   }
 
   getHumanReadableVisibility(value) {
-    return value ? "Public" : "Hidden";
+    return value == "true" ? "Public" : "Hidden";
```

Comment:
Because it comes from the aggregations of ES, and it's defined as a string.

![image](https://user-images.githubusercontent.com/15194802/169285857-82055812-2393-4557-862f-293aade7fe0e.png)